### ISSUE TITLE
test: degradation on 1.10 after commit 4c0fe801 "test: update test-run"

### ISSUE THREAD

**Issue by avtikhon** [tarantool/tarantool#5069]
Tarantool version:
1.10
4c0fe801 
"test: update test-run"

OS version:
All

Bug description:
Found new issues in pipeline jobs:
https://gitlab.com/tarantool/tarantool/-/commit/4c0fe80114e3ecd37288cd41335a07c7dcf4359d/pipelines?ref=1.10

Issues:
1. small/rlist.test
```
 [132] small/rlist.test                                                
 [132] TAP13 parse failed (Missing plan in the TAP source).
 [132] 
 [132] No result file (small/rlist.result) found.
 [132] Run the test with --update-result option to write the new result file.
 [132] [ fail ]
```
2. small/static.test
```
 [132] small/static.test                                               
 [132] TAP13 parse failed (Missing plan in the TAP source).
 [132] 
 [132] No result file (small/static.result) found.
 [132] Run the test with --update-result option to write the new result file.
 [132] [ fail ]
```

Steps to reproduce:

Optional (but very desirable):
* coredump
* backtrace
* netstat

**Comment by Totktonada** [comment]
There are no small/rlist.test and small/static.test on 1.10. Please, verify CI setup.

Looks related to #4258 and maybe #5036.

**Comment by avtikhon** [comment]
Right, the issue depends on gitlab-ci cleanup, let's fix it within #5036 issue.
